app/furniture/marketplace/management_component.rb

```diff
@@ -7,5 +7,17 @@ def initialize(marketplace:, **kwargs)
 
       self.marketplace = marketplace
     end
+
+    def button_to(location, icon:)
```

Comment:
This is probably ok, but it could lead to some confusion, since it is shadowing the built-in Rails `button_to` helper.

Makes me wonder if what we want is to extend the `button_to` helper to take some arguments to use our `ButtonComponent` instead or in addition to this change.

Reply:
Yea, I think there's definitely a risk there; especially since in this case we need to also call `render`; where is in classic rails `button_to` does not require a render. 

My instincts are that it would be nice to start replace the built-in rails helpers (including the form-builder) so that folks who are coming straight from rails don't have to deal with the bunny-hop...

Reply:
I renamed to `button` for now; since `button_to` also presumes to make a form; and I didn't want to deal with that.